src/Workspaces/Core/Portable/Options/EditorConfig/EditorConfigDocumentOptionsProviderFactory.cs

```diff
@@ -30,10 +29,13 @@ public IDocumentOptionsProvider TryCreate(Workspace workspace)
             return new EditorConfigDocumentOptionsProvider(workspace.Services.GetService<IErrorLoggerService>());
         }
 
+        public static readonly Option<bool> UseLegacyEditorConfigSupport =
+            new Option<bool>(nameof(EditorConfigDocumentOptionsProviderFactory), nameof(UseLegacyEditorConfigSupport), defaultValue: false,
+                storageLocations: new LocalUserProfileStorageLocation("Roslyn\\.editorconfig\\UseLegacySupport"));
```

Comment:
This should go in `InternalFeatureOnOffOptions.LocalRegistryPath`:

```suggestion
                storageLocations: new LocalUserProfileStorageLocation(InternalFeatureOnOffOptions.LocalRegistryPath + "LegacyEditorConfig"));
```